src/tokens/transformation/tailwind-email/formatTokens.ts

```diff
@@ -0,0 +1,159 @@
+import merge from 'lodash.merge'
+import { Formatter } from 'style-dictionary'
+
+const themes = ['light', 'dark']
+
+const kebabCase = (str: string) => str && str.toLowerCase().replaceAll(' ', '-')
+
+/**
+ * This function transforms tokens into a nested object
+ * structure ready for Tailwind. The conditional statements
+ * are largely to handle creating both static and dynamic
+ * tokens. E.g. A given token needs three variants:
+ * dynamic, static light, and static dark. This function gets
+ * run twice: once to create all static tokens, and another
+ * time to create the dynamic tokens which places values on
+ * the parent (e.g. the root object, or 'legacy') and uses the
+ * appropriate color variable.
+ */
+function createColorTokensFromGroup(tokens, staticTheme = true) {
+  const colorTokens = {}
+  tokens.forEach(({ type, name, ...t }) => {
```

Comment:
optional nit: I think its generally nicer/more readable/faster to use a `for .. of` loop